manager/allocator/network.go

```diff
@@ -696,6 +714,32 @@ func (a *Allocator) allocateTask(ctx context.Context, nc *networkContext, tx sto
 	return storeT, nil
 }
 
+func (a *Allocator) procUnallocatedNetworks(ctx context.Context, nc *networkContext) {
+	for _, n := range nc.unallocatedNetworks {
+		if !nc.nwkAllocator.IsAllocated(n) {
+			if err := a.allocateNetwork(ctx, nc, n); err != nil {
+				log.G(ctx).Errorf("Failed allocation of unallocated network %s: %v", n.ID, err)
```

Comment:
Will this generate a lot of log noise if a network wasn't allocated at first and still can't be allocated afterwards? We'll try every time there's a commit event, and produce a log message every time.